### DISCUSSION TITLE
all the coins were stolen from my  wallet

### DISCUSSION THREAD

**Discussion by jefferyjgy** [Chia-Network/chia-blockchain#14751] [Support]
Here are the details：

<html>
<body>

xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tduguConfirmed | March 5, 2023 7:33 PM | - 0.000000000099 XCH | 0 XCH |  
-- | -- | -- | -- | --
  | Confirmed at Height3335975MemosNot Available | Confirmed at Height | 3335975 | Memos | Not Available
Confirmed at Height | 3335975
Memos | Not Available
  | xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tduguConfirmed | March 5, 2023 3:41 PM | - 2.000615000994 XCH | 0 XCH |  
  | Confirmed at Height3335255MemosNot Available | Confirmed at Height | 3335255 | Memos | Not Available
Confirmed at Height | 3335255
Memos | Not Available
  | xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tduguConfirmed | March 5, 2023 1:46 PM | - 11.27486370311 XCH | 0 XCH |  
  | Confirmed at Height3334918MemosNot Available | Confirmed at Height | 3334918 | Memos | Not Available
Confirmed at Height | 3334918
Memos | Not Available
  | xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tduguConfirmed | March 5, 2023 1:09 PM | - 0.00008557998 XCH | 0 XCH |  
  | Confirmed at Height3334782MemosNot Available | Confirmed at Height | 3334782 | Memos | Not Available
Confirmed at Height | 3334782
Memos | Not Available
  | xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tduguConfirmed | March 5, 2023 12:19 PM | - 1.943270016972 XCH | 0 XCH |  
  | Confirmed at Height3334619MemosNot Available | Confirmed at Height | 3334619 | Memos | Not Available
Confirmed at Height | 3334619
Memos | Not Available

</body>
</html>

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
